src/stacked_hourglass/utils/transforms.py

```diff
@@ -78,38 +86,52 @@ def transform(pt, center, scale, res, invert=0, rot=0):
 
 
 def transform_preds(coords, center, scale, res):
-    # size = coords.size()
-    # coords = coords.view(-1, coords.size(-1))
-    # print(coords.size())
     for p in range(coords.size(0)):
         coords[p, 0:2] = to_torch(transform(coords[p, 0:2], center, scale, res, 1, 0))
     return coords
 
 
 def crop(img, center, scale, res, rot=0):
+    """
+    img: torch.Tensor: shape: C,H,W
+    center: torch.Tensor: shape: x, y
+    scale: [H/200, W/200]
+    res: H,W
+    """
+    # If given 1D scale as a torch Tensor, convert to a pair of scales
```

Comment:
I'm worried about the robustness of this code. How about:

```python
scale = np.asarray(scale).flatten()
if len(scale) == 1:
    scale = scale.repeat(2)
```

Reply:
That's a great suggestion! This will be a nice robust solution